ansible/playbooks/AdoptOpenJDK_Unix_Playbook/main.yml

```diff
@@ -131,3 +131,5 @@
     - IPv6
     - role: disable_gui
       tags: adoptopenjdk
+    - role: logs
+      tags: always
```

Comment:
tags: always is not really needed if it is intended to run all the time.
unless you want to use "--skip-tags always" in certain case?

Reply:
Without `tags: always`, it will still run when the playbook is run as a whole, but not when the playbook is run with the `--tags` flag (where you specify only certain roles to run). The `always` tag ensures it runs in both circumstances